frontend/src/Components/Utilities/FilterInterface/FilterBoard.tsx

```diff
@@ -215,9 +215,10 @@ function FilterBoard() {
                   value={surgeonCasesPerformed}
                   onChange={(event, value) => { store.configStore.changeSurgeonCasesPerformed(value as [number, number]); }}
                   marks={[
-                    0,
+                    defaultState.surgeonCasesPerformed[0],
                     Math.max(...Object.values(store.surgeonCasesPerformedRange)),
                   ].map((d) => ({ value: d, label: d.toString() }))}
+                  min={defaultState.surgeonCasesPerformed[0]}
```

Comment:
We should still allow 0
```suggestion
                  min={0}
```